src/DeviceListener.ts

```diff
@@ -353,14 +377,19 @@ export default class DeviceListener {
                     crossSigningStatus.privateKeysCachedLocally,
                 );
                 showSetupEncryptionToast(SetupKind.KEY_STORAGE_OUT_OF_SYNC);
+            } else if (!keyStorageIsOk) {
+                logSpan.info("Key storage is turned off: showing TURN_ON_KEY_STORAGE toast");
```

Comment:
the fact that we talk about "key storage" here but "backup" just below is going to make it confusing to grok these logs. I'd prefer we stick with the term "backup" internally if we're just talking about message key storage.